### DISCUSSION TITLE
Using multiple org.redisson.api.RedissonClient in singe application

### DISCUSSION THREAD

**Discussion by gargravi** [redisson/redisson#5656] [General]
I have requirement where I need to connect to different deployment of Redis from a single java application. I'll appreciate help on following queries
1. Can I use two instance of RedissonClient in my java application (I am not sure if within redisson library any configuration or internal object has global/static footprint which may cause issues in using multiple instances of `org.redisson.api.RedissonClient`)
2. If answer to above is yes, then can redisson library support connecting to following two deployments within the same application
     a) One Redis cluster and another Redis-enterprise
     b) Two different deployment of Redis-enterprise

**Comment by mrniko** [comment]
> Can I use two instance of RedissonClient in my java application

Yes. You need to create a separate Redisson configuration per Redis deplyment and use two RedisClient instances.

> If answer to above is yes, then can redisson library support connecting to following two deployments within the same application

Yes

**Reply by gargravi** [reply]
@mrniko Thanks for your response.
